drivers/ec_host_cmd_periph/ec_host_cmd_periph_espi.c

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2022 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT zephyr_ec_host_cmd_periph_espi
+
+#include <string.h>
+#include <zephyr/device.h>
+#include <zephyr/drivers/espi.h>
+#include <zephyr/mgmt/ec_host_cmd.h>
+#include <zephyr/drivers/ec_host_cmd_periph/ec_host_cmd_periph.h>
+
+#if !DT_NODE_EXISTS(DT_DRV_INST(0))
+#error The compatible node must exist and select the eSPI bus
+#endif
+
+#define ESPI_BUS DT_PHANDLE(DT_DRV_INST(0), bus)
+#define ESPI_DEVICE DEVICE_DT_GET(ESPI_BUS)
+
+struct ec_host_cmd_periph_espi_data {
+	struct k_sem handler_owns;
+	struct k_sem dev_owns;
+	size_t rx_buffer_len;
```

Comment:
This is just assigned to a constant right? Should prob go in a _cfg structure (or go entirely)

Reply:
Now it will be read from the eSPI subsystem, so it must be in the data.